# HSV
HSV_RANGE/H_MIN: 121
HSV_RANGE/H_MAX: 228
HSV_RANGE/S_MIN:  76
HSV_RANGE/S_MAX: 255
HSV_RANGE/V_MIN:  43
HSV_RANGE/V_MAX: 199

USE_MORPHOPS: true
USE_TRACK: true
MAX_NUM_OBJECTS: 1
MIN_OBJECT_AREA: 1000
MAX_OBJECT_AREA: 307200

# debug
DEBUG/IS_SHOW: false
DEBUG/USE_TRACKBAR: false


